app/controllers/people_skills_controller.rb

```diff
@@ -31,4 +31,20 @@ def extract_required_search_values
     filtered_array = filter_params.skill_ids.zip(filter_params.levels, filter_params.interests)
     filtered_array.reject { |arr| arr.first == '' }.transpose
   end
+
+  def find_skill_title_by_id(id)
+    Skill.find(id).title
+  end
+
+  def get_no_match_translation(index)
```

Comment:
Try to work on your naming as well.
`get_no_match_translation` doesn't really speak for itself. Especially because it handles several cases at the same time (no skill was found / several skills weren't found) Also the `get` part doesn't help with anything, you call the method so getting is implied. 

Something like `no_results_message` covers them all, says what it actually is and is more readable.